cmd/controller/app/options/options.go

```diff
@@ -49,6 +55,12 @@ func (o *ControllerRunOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&o.EnableLeaderElection, "enable-leader-election", o.EnableLeaderElection,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	fs.IntVar(&o.LeaseDuration, "lease-duration", o.LeaseDuration,
```

Comment:
Can we use `fs.DurationVar` instead of `fs.IntVar`, it will save the convert process.

```
fs.DurationVar(&o.LeaseDuration, "lease-duration", o.LeaseDuration,
	"Non-leader candidates will wait to force acquire leadership. Default is 15 seconds.")
```

Reply:
> Can we use `fs.DurationVar` instead of `fs.IntVar`, it will save the convert process.
> 
> ```
> fs.DurationVar(&o.LeaseDuration, "lease-duration", o.LeaseDuration,
> 	"Non-leader candidates will wait to force acquire leadership. Default is 15 seconds.")
> ```

Thank you.
I did not notice this func previously, changed.

Reply:
NP！ looks good to me now👍